### PR TITLE
Adapt to new payson web page

### LINKED ISSUE #145: Payson is no longer working

### PR DESCRIPTION
**Pull request by robho**
Payson was broken in bankdroid because of changes to payson's web page. Adapt. Fixes #145.

Support for listing transactions is not yet implemented. The transactions
are no longer available in json format, but must instead be parsed from HTML.

### CODE REVIEW

File: bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Payson.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        if (username == null || password == null || username.length() == 0 || password.length() == 0) {
+            throw new LoginException(res.getText(R.string.invalid_username_password).toString());
+        }
+        urlopen = login();
+
+        try {
+            Account account = new Account(res.getText(R.string.balance).toString(),
+                    Helpers.parseBalance(userInfo.getString("balance")), "1");
+            accounts.add(account);
+            balance = balance.add(account.getBalance());
```
Comment (fredrike) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Payson.java]: You don't need to set balance, it's automatically calculated.

Comment (robho) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/Payson.java]: Thanks, that's good to know. Pull request updated.

**Comment by goober** [comment]
Thanks for the pull request. version `1.9.8.0` is feature complete and soon to be shipped to production. I will accept this in the next version.

**Comment by fredrike** [comment]
Thank you! 

Looking forward for a patch covering transactions as well.

**Pull request merged**